core-blocks/archives/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ArchivesBlock from './block';
+export const name = 'core/archives';
+
+export const settings = {
+	title: __( 'Archives' ),
+
+	description: __( 'This block displays a monthly archive of your site’s Posts.' ),
+
+	icon: 'calendar-alt',
+
+	category: 'widgets',
+
+	keywords: [ __( 'archives' ) ],
+
+	supports: {
+		html: false,
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( 'left' === align || 'right' === align || 'center' === align ) {
+			return { 'data-align': align };
+		}
+	},
+
+	edit: ArchivesBlock,
+
+	save() {
+		// Handled by PHP.
+		return null;
```

Comment:
Is there something we can return here as a fallback for clients like RSS readers that don't support dynamic blocks? 

(Probably not, just wondering.)